astropy/coordinates/tests/test_solar_system.py

```diff
@@ -212,23 +213,6 @@ def test_de432s_planet(self, body):
             astropy.distance, horizons.distance, atol=de432s_distance_tolerance
         )
 
-    @pytest.mark.remote_data
-    @pytest.mark.skipif(not HAS_JPLEPHEM, reason="requires jplephem")
-    def test_de432s_moon(self):
-        astropy = get_moon(self.t, ephemeris="de432s")
-        horizons = self.horizons["moon"]
-
-        # convert to true equator and equinox
-        astropy = astropy.transform_to(self.apparent_frame)
-
-        # Assert sky coordinates are close.
-        assert astropy.separation(horizons) < de432s_separation_tolerance_moon
```

Comment:
Note that the definition of `de432s_separation_tolerance_moon` can also be removed (and it is the same as for planets, so the tests indeed stay the same)

Reply:
I've removed `de432s_separation_tolerance_moon`.

Reply:
@mhvk, I've implemented your suggestion. Can this be merged now?